src/Accessibility/AXAnimatedImagesUtilities.cs

```diff
@@ -0,0 +1,18 @@
+#nullable enable
+
+using System;
+using System.Runtime.InteropServices;
+using CoreGraphics;
+using Foundation;
+using ObjCRuntime;
+
+namespace Accessibility {
+
+	public static partial class AXAnimatedImagesUtilities {
+
+		[DllImport (Constants.AccessibilityLibrary)]
+		extern static byte AXAnimatedImagesEnabled ();
+
+		public static bool Enabled => AXAnimatedImagesEnabled () != 0;
```

Comment:
No availability attributes?

Reply:
No need, they are in the class in the accessibility.cs